### PR TITLE
Add System.ComponentModel library

### PR DESCRIPTION
**Pull request by YingP99**
Add System.ComponentModel library and test

### CODE REVIEW

File: src/System.ComponentModel/src/System.ComponentModel.csproj [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (joshfree) [inline src/System.ComponentModel/src/System.ComponentModel.csproj]: This should be 12

Comment (joshfree) [inline src/System.ComponentModel/src/System.ComponentModel.csproj]: Also I see the same bug in System.Collections.Concurrent.csproj - please fix that file in another change too

**Comment by stephentoub** [comment]
LGTM

**Comment by weshaggard** [comment]
LGTM Thanks @YingP99

**Pull request merged**